transport-native-epoll/src/main/java/io/netty/channel/epoll/TcpMd5Util.java

```diff
@@ -40,14 +41,7 @@
                 throw new IllegalArgumentException("newKeys contains an entry with null address: " + newKeys);
             }
             requireNonNull(key, "newKeys[" + e.getKey() + "]");
-            if (key.length == 0) {
-                throw new IllegalArgumentException("newKeys[" + e.getKey() + "] has an empty key.");
-            }
-            if (key.length > Native.TCP_MD5SIG_MAXKEYLEN) {
-                throw new IllegalArgumentException("newKeys[" + e.getKey() +
-                    "] has a key with invalid length; should not exceed the maximum length (" +
-                        Native.TCP_MD5SIG_MAXKEYLEN + ')');
-            }
+            checkClosedInterval(key.length, 1, Native.TCP_MD5SIG_MAXKEYLEN, "newKeys[" + e.getKey() + "]");
```

Comment:
This is string formatting as well. Although apparently it was already computed once above? Still not a great idea.